README.md

```diff
@@ -61,3 +58,96 @@ of the event.
+issues until they are happy with their work!
+
+At the middle of the spree, the Scala Center and sponsors of the event provide
+maintainers and participants alike with some refreshment (drinks, sandwiches,
+pizza, etc).
+
+Participants can leave the event at any time they want. When the time approaches
+the end, everyone starts to wrap up: participants finish their PRs while
+maintainers finish their review, and organizers of the spree give away Scala
+t-shirts. We finish by thanking your hard work for open-source.
+
+## How to propose a project
+
+A Scala Center spree is the perfect event to gauge interest in your open-source
+projects. You not only have the opportunity to get new contributors involved in
+your project, but you can win friends and lifetime maintainers that help you
+make a difference in the open-source world.
+
+There is only one requirement to submit a project -- you need to be present for
+the duration of the Scala Center spree. Your physical presence is important to
+assist and motivate contributors.
+
+If you are a maintainer of an open-source Scala project, make a PR to add your
+project to the list!
+
+### What you need to do
+
+Create a PR with the following information:
+  
+* Project information.
+* Your contact details.
+* A link to a "spree" or "hackathon" label with curated tickets for the
+  participants. See [this project](https://github.com/sbt/zinc/issues?utf8=✓&q=label:hackathon%20is:issue) for inspiration.
+
+It's important that the curated tickets are entry-level, typical issues that you
+would solve in 15-20 minutes of your time as an experienced maintainer. In our
+experience, newcomers will take 1 to 2 hours to address them, assuming they are
+unfamiliar with the codebase and this is their first contribution.
+
+#### How to be an effective maintainer
+
+Maintainers can make a difference by tweaking some knobs:
+  
+* Provide a `CONTRIBUTING` guide. Contributing guides explain to newcomers the
+    usual workflow to get started and what's expected from them. Good guides:
+    [scala/scala](https://github.com/scala/scala/blob/2.12.x/CONTRIBUTING.m://github.com/scala/scala/blob/2.12.x/CONTRIBUTING.md),
+    [sbt/zinc](https://github.com/sbt/zinc/blob/1.0/CONTRIBUTING.md), [scalameta/scalameta](https://github.com/scalameta/scalameta/blob/master/CONTRIBUTING.md),
+		[scalacenter/scalafix](https://github.com/scala/scala/blob/2.12.x/CONTRIBUTING.md).
+* Provide a motivating `README` with clear instructions. Make sure docs are up to date.
+* Link to documentation when possible, or show contributors how to search for
+    docs. You can label as `docs` any issue or PR with relevant discussions to
+    get acquainted with implementation details and design decisions.
+* [Be nice to newcomers](http://brson.github.io/2017/04/05/minimally-nice-maintainer), they
+    will always remember it!
+
+#### How to write good tickets
+
+Curating tickets is not easy. If you want to optimize for the highest number of
```

Comment:
I think you misunderstood my question. My point was that this section seemed like general ticket writing advice, rather than something specifically for the sprees.